pallav/fashionTech/pallavBanka/parkingAssignment/Parking.java

```diff
@@ -1,2 +1,173 @@
-package com.fashionTech.pallavBanka.parkingAssignment;
+package fashionTech.pallavBanka.parkingAssignment;
 
+import java.util.Date;
+import java.util.Timer;
+
+abstract class parking{
+    void parkVehicle(vehicle v){
+        int regno;
+        short floor;
+        int slotid;
+        regno=v.regno;
```

Comment:
line 8 & 11 could have been done in single statement.
Also what is the point of assigning method local variables some value, when it is not returned outside
Use Integer instead of int/ Short instead of short...avoid primitive types.
v.regno should not be accessible, you should have setters/getters